src/components/phone/AgentAnalyticsCard.vue

```diff
@@ -44,10 +44,10 @@ export default {
     },
     lang() {
       const metricLang = {};
-      metricLang[METRICS.AGENTS_ON_CALL] = '~~On the Phone Right Now';
-      metricLang[METRICS.ONLINE] = '~~Volunteers Online';
-      metricLang[METRICS.CONTACTS_QUEUED] = '~~Peopled in Queue';
-      metricLang[METRICS.NEEDED] = '~~Volunteers Needed';
+      metricLang[METRICS.AGENTS_ON_CALL] = this.$t('agentAnalyticCard.on_the_phone');
+      metricLang[METRICS.ONLINE] = this.$t('agentAnalyticCard.volunteers_online');
+      metricLang[METRICS.CONTACTS_QUEUED] = this.$t('agentAnalyticCard.people_in_queue');
+      metricLang[METRICS.NEEDED] = this.$t('agentAnalyticCard.volunteers_needed');
```

Comment:
Drop this `this.$t` part off of these please ;)

The `this.getLang` call below handles that automagically